### DISCUSSION TITLE
disableRegex seemingly not working

### DISCUSSION THREAD

**Discussion by jimhranicky** [Security-Onion-Solutions/securityonion#13698] [2.4] [answered]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

64g

### Storage for /

80g

### Storage for /nsm

2T

### Network Traffic Collection

tap

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have

  ET.*PHISHING.*

added 
![Screenshot_20240918_204853](https://github.com/user-attachments/assets/379089dd-5288-450e-a528-76d6e70a870f)
under

  soc > config > server > modules > suricataengine > disableRegex

However, it seems the rules aren't getting commented out?

From one of our sensors: 

  egrep '^alert.*ET PHISH' /opt/so/conf/suricata/rules/all.rules | wc -l
  238

Did I miss a step?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
Have you tried using the regex syntax shown at https://docs.securityonion.net/en/2.4/nids.html#enabling-and-disabling-with-regex?

**Reply by jimhranicky** [reply]
Yes, added this: 

ET PHISHING\s

but still: 

% egrep '^alert.*ET PHISH' /opt/so/conf/suricata/rules/all.rules | wc -l
238

on one of our sensors

**Comment by defensivedepth** [comment]
@jimhranicky Have you seen alerts from these rules?

**Reply by jimhranicky** [reply]
I'm primarily trying to reduce load on the suricata engine - does the disableregex only suppress alerts, or is it supposed to disable the rules entirely?

**Reply by defensivedepth** [reply, marked as answer]
It disables them entirely. For rules with flowbit dependencies, they are still enabled but set to `noalert`. Those are the rules you are seeing.
